src/applications/gi-sandbox/containers/search/LocationSearchResults.jsx

```diff
@@ -99,19 +100,62 @@ function LocationSearchResults({
     markers.current.push(markerElement);
   };
 
+  const getCurrentLocationCenter = () => {
+    let center = '';
+    if (streetAddress.searchString) {
+      center = new mapboxgl.LngLat(
+        streetAddress.position.longitude,
+        streetAddress.position.latitude,
+      );
+    }
+    return center;
+  };
+
+  const getZoomLevel = distance => {
```

Comment:
this is not a good way to do this
https://docs.mapbox.com/help/glossary/zoom-level/#zoom-levels-and-geographical-distance

Given the difficulties with centering and zooming the centering on the user's location should have been discussed as this issue was also discovered when doing a search using the input